community/bitcoin/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=bitcoin
-pkgver=0.16.1
+pkgver=0.16.2
 _ver=${pkgver/_/}
 pkgrel=3
```

Comment:
sadly I missed `pkgrel` bump

Reply:
`$pkgrel` fixed in https://github.com/alpinelinux/aports/pull/4939